index.css

```diff
@@ -0,0 +1,182 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Lato", sans-serif;
+}
+
+.container {
+  font-family: 'Lato', sans-serif;
```

Comment:
Good job styling the font family of your webpage :+1: in my opinion, it could be a lot better off removing this font family because it is already put for the whole webpage on line 5 of your styles, what do you think?